modules/ROOT/pages/syntax/naming.adoc

```diff
@@ -62,8 +62,8 @@ Some techniques to mitigate this are:
 Here are the recommended naming conventions:
 
 |===
-| Node labels          | Camel-case, beginning with an upper-case character | `:VehicleOwner` rather than `:vehicle_owner` etc.
-| Relationship types   | Upper-case, using underscore to separate words     | `:OWNS_VEHICLE` rather than `:ownsVehicle` etc.
+| Node labels          | Pascal case, which is similar to camel case but beginning with an upper-case character | `:VehicleOwner` rather than `:vehicle_owner` etc.
```

Comment:
```suggestion
| Node labels          | Pascal Case, Capitalized words using no spaces or separators | `:VehicleOwner` rather than `:vehicle_owner` etc.
```